packages/core/src/lib/store.ts

```diff
@@ -62,9 +76,16 @@ const reducer = (
         return state;
       }
 
+      const accountStates = accounts.map((account, i) => {
+        return {
+          ...account,
+          active: i === 0,
```

Comment:
This scenario is a little more involved as we should try to maintain the existing account that's selected unless it's been removed (in which case we would default to the first one)